.gitignore

```diff
@@ -32,7 +32,6 @@ coverage.xml
 .tox
 .mypy_cache
 .pytest_cache
-nosetests.xml
```

Comment:
why did you remove it from .gitignore? The file isn't part of this commit.

Reply:
Because we are not using nosetests in this project.

Reply:
I noticed it while I was fixing flake8 issues.

Reply:
Is that ok, @bastelfreak?

Reply:
Thank for the approval, @bastelfreak !